src/backend/columnar/columnar_writer.c

```diff
@@ -129,6 +130,7 @@ ColumnarBeginWrite(RelFileNode relfilenode,
 	writeState->comparisonFunctionArray = comparisonFunctionArray;
 	writeState->stripeBuffers = NULL;
 	writeState->stripeSkipList = NULL;
+	writeState->stripeFirstRowNumber = 0;
```

Comment:
we will anyway set `writeState->stripeFirstRowNumber` in `ColumnarWriteRow` before using, but it still might be better to use `COLUMNAR_FIRST_ROW_NUMBER` here

Reply:
now we set it to COLUMNAR_INVALID_ROW_NUMBER here